### DISCUSSION TITLE
Add a way to account for Xp boosters?

### DISCUSSION THREAD

**Discussion by NamiestOfNames** [xanthics/gw2craft#66] [Ideas]
There could be a seperate text box at the top where you could enter the boost xp % you plan to have while crafting, and then the program could decrease what you need by that %. This is just an idea, I'm not sure how possible it is.

**Comment by xanthics** [comment]
Are there xp boosters that work with crafting xp now? Can you link the specific items from the wiki? The only think I am aware of that would change the formula is production downtime. Also a limited possibility of 100% crit, but wasn't permanently available last I checked.

**Comment by NamiestOfNames** [comment]
On second look it seems I was mistaken. Although the booster enhanced crafting experience, it was player experience for crafting rather than CXP. My bad.
